scripts/publish.js

```diff
@@ -15,10 +17,20 @@ yargs.scriptName('publish.js').usage('$0 [args] [explicitPackage1...] [explicitP
 
 const explicitPackages = yargs.argv._;
 
-process.cwd(__dirname);
 const p = (str) => process.stdout.write(str);
-const files = fs.readdirSync(process.cwd());
-const pkgs = files.filter((file) => fs.statSync(file).isDirectory() && fs.existsSync(path.join(file, 'package.json')));
+const getPackagePath = (pkg) => path.resolve(`${MODULES_DIR}/${pkg}`);
```

Comment:
NBD but I generally try to use path.resolve or path.join everywhere in case windows devs exist